streamwebs_frontend/streamwebs/views.py

```diff
@@ -85,7 +49,6 @@ def register(request):
             profile.user = user
             profile.save()
             registered = True
-
         else:
```

Comment:
To get rid of the garbage text that pops up whenever you run tests, you can delete this else statement/print statement :)